src/OSPSuite.R/MinimalImplementations/RExecutionContext.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using OSPSuite.Core.Commands;
+using OSPSuite.Core.Commands.Core;
+using OSPSuite.Core.Domain;
+
+namespace OSPSuite.R.MinimalImplementations
+{
+   public class RExecutionContext : IOSPSuiteExecutionContext
```

Comment:
A little bit unclean. But for the SA, we need th execution context that is now quite substantial. Defining a defualt one here